assets/scss/_boot.scss

```diff
@@ -10,6 +10,8 @@
 @import "./layout/header";
 @import "./layout/header-about-the-studio";
 @import "./layout/header-logotypes-subpage";
+@import "./layout/header-project-subpage";
+@import "./layout/header-title-number-one-subpage";
```

Comment:
to jest chyba podstrona poj. projektu, tak?
Jak tak to - header-signle-project.html